dart/test/utils_test.dart

```diff
@@ -23,8 +23,8 @@ void main() {
     });
 
     test('enum returns described enum during serialization', () {
-      expect(
-          jsonSerializationFallback(SentryOrientation.landscape), 'landscape');
+      expect(jsonSerializationFallback(SentryOrientation.landscape),
+          'SentryOrientation.landscape');
```

Comment:
for known SDK types this is not a problem
See https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/protocol/sentry_device.dart#L178-L194
this is only for unknown types set dynamically by the user.